doc/packer.txt

```diff
@@ -126,6 +127,14 @@ Perform `PackerUpdate` and then `PackerCompile`.
 `PackerLoad`                                    *packer-commands-load*
 Loads opt plugin immediately
 
+`PackerSnapshot`                                    *packer-commands-snapshot*
```

Comment:
This should probably be fixed for the other commands too (not as part of this PR), but we might want to specify the arguments accepted for the snapshot related commands (i.e. the snapshot file).